Update Ecto.Multi.run example

darksheik opened the pull request:
My jobs did not run initially after upgrading to the latest release, as it seems the new spec of `Ecto.Multi.run` is now that the 3rd arg now is a function with arity 2.  (current Ecto is 3.1.4)

Updating example in `ecto_job` doc to reflect that change.

From `Ecto.Multi`: 
```
[Ecto.Multi] Ecto.Multi.run/3 now receives the repo in which the transaction is executing as the first argument to functions, and the changes so far as the second argument
```

Code:
```
  @doc """
  Adds a function to run as part of the multi.

  The function should return either `{:ok, value}` or `{:error, value}`,
  and receives the repo as the first argument, and the changes so far
  as the second argument.

  ## Example

      Ecto.Multi.run(multi, :write, fn _repo, %{image: image} ->
        with :ok <- File.write(image.name, image.contents) do
          {:ok, nil}
        end
      end)
  """
  @spec run(t, name, run) :: t
  def run(multi, name, run) when is_function(run, 2) do
    add_operation(multi, name, {:run, run})
  end
```

darksheik committed:
Update Ecto.Multi.run example

VitorTrin commented on README.md:
```diff
@@ -125,7 +125,7 @@ defmodule MyApp.JobQueue do
   def perform(multi = %Ecto.Multi{}, job = %{"type" => "SendEmail", "recipient" => recipient, "body" => body}) do
     multi
-    |> Ecto.Multi.run(:send, fn _ -> EmailService.send(recipient, body) end)
+    |> Ecto.Multi.run(:send, fn _, _ -> EmailService.send(recipient, body) end)
```
Considering this is documentation, what about
```suggestion
    |> Ecto.Multi.run(:send, fn _repo, _changes -> EmailService.send(recipient, body) end)
```
?

darksheik committed:
Update README.md

Sounds good to me.  Some folks might want to use that additional info, good to know at a glance.

Co-Authored-By: Vítor Luís dos Santos Trindade <emailreservadovitor@gmail.com>

mbuhot commented:
Thanks @darksheik @VitorTrin 👍
